### PR TITLE
Workbot shortcuts

### PR DESCRIPTION
**Pull request by jandonyada**
### Which issues does this fix?
Fixes #12345

### PR submission checklist
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [x] Added new files to `docs/vuepress/sidebar.js`
- [x] Check for broken links
```bash
npm run check
```

### CODE REVIEW

File: docs/workbot/workbot-shortcuts.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+![Message shortcut gif](~@img/workbot/workbot-message-actions/message-actions-example.gif)
+*Message shortcut in action*
+
+Similar to the <b>[Post command](/workbot/workbot-commands.md)</b> trigger, command input fields can be defined if you need user input for follow-up actions. If command input fields are specified, shortcuts will launch a dialog to get user input.
+
+## How global shortcuts work
+Shortcuts can be triggered from the shortcuts menu or the search bar.
+
+Click on the shortcuts menu or search for your shortcut in the search bar at the top of your Slack UI.
+s
```
Comment (eeliang) [inline docs/workbot/workbot-shortcuts.md:24]: ~s~

File: docs/workbot/workbot-shortcuts.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+![Accessing shortcuts](~@img/workbot/workbot/shortcuts-example.jpg)
+*Accessing shortcuts from Slack*
+
+## How message shortcuts work
+Message shortcuts can be triggered on any message in any channel, direct message, or multi-party message that your custom bot is a member of.
+
+Hover on a message, and click on the horizontal ellipsis menu to bring up a list of actions. Message shortcuts that you've created can be found at the bottom of the list.
```
Comment (eeliang) [inline docs/workbot/workbot-shortcuts.md:31]: Hover on a message ~,~ (remove comma) and click...

File: docs/workbot/workbot-shortcuts.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+            The name of the shortcut should be descriptive of its function, e.g. Create Jira ticket. <br><br>The shortcut name also doubles as a <b>Callback ID</b>, which is used in configuring the corresponding shortcut in Slack.<br><br>Shortcut names should be unique across all recipes.
+          </td>
+        </tr>
+        <tr>
+          <td>Command input fields</td>
+          <td>
+            Fields you need from the user
+          </td>
+          <td>
+          Often, you need your bot to obtain more information from users before it can carry out subsequent recipe actions. <br><br>If you've defined any command input fields, the custom bot will automatically launch a dialog for users to key in each input field. Define up to 10 command input fields.
```
Comment (eeliang) [inline docs/workbot/workbot-shortcuts.md:87]: How many command input fields are allowed? Is it 10 or 5 command input fields? (line 87, line 113)

File: docs/workbot/workbot-shortcuts.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+You can choose from 3 different types of select menus. Each differs by the menu options displayed:
+  - **Channels** displays the list of all public channels in your Slack workspace.
+
+  ![Channel menu options](~@img/workbot/workbot-message-actions/channels-select.gif)
+  *Example of a channels menu in a Slack dialog*
+
+  When a channel is selected, the channel ID is returned, e.g. **CA83MDSK0**.
+
+  - **users** displays the list of all users (including bots) in your Slack workspace.
```
Comment (eeliang) [inline docs/workbot/workbot-shortcuts.md:156]: Users (caps)

File: docs/workbot/workbot-shortcuts.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+
+![Create new shortcut](~@img/workbot/workbot/create-new-shortcut.png)
+*Click on **Create New Shortcut***
+
+On the next page, choose **Global**.
+
+![Choose Global](~@img/workbot/workbot/choose-global.png)
+*Click on **Global***
+
+In the shortcuts configuration, choose a descriptive name for your shortcut, a short description of its function, followed by the **Callback ID**.
```
Comment (eeliang) [inline docs/workbot/workbot-shortcuts.md:200]: Use an anchor \[\*\*Callback ID**\]\(#callback-id).

File: docs/workbot/workbot-shortcuts.md (hunk 3 above)
Comment (jandonyada) [reply docs/workbot/workbot-shortcuts.md:87]: it's 10, nice catch

**Pull request merged**